### PR TITLE
Entity QC: Zones 001 - 005

### PR DESCRIPTION
**Pull request by wrenffxi**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Some mob and npc QC for zones 001 - 005.  Compared what's in our DB against Nyu's retail captures.

Of note, several of our ??? NPCs were mixed up compared to retail, id-vs-position-wise.  In this PR I have corrected the QM positions, and also changed the names from qm# to something that describe what the QM does/spawns, such as qm_cryptonberries.  Numerically-ordered QMs fail when SE moves, adds, and removes NPCs.

### CODE REVIEW

**Comment by zach2good** [comment]
Half-reviewed this, all looking good so far 👍 

> ... also changed the names from qm# to something that describe what the QM does/spawns, such as qm_cryptonberries. Numerically-ordered QMs fail when SE moves, adds, and removes NPCs.

This is a change I deeply appreciate!

File: sql/mob_spawn_points.sql [hunk 1]
```diff
@@ -117,904 +117,910 @@ INSERT INTO `mob_spawn_points` VALUES (16785462,'Land_Pugil','Land Pugil',17,-31
...
+INSERT INTO `mob_spawn_points` VALUES (16793707,'Alraune','Alraune',13,-251.526,-3.930,-527.831,156);
+INSERT INTO `mob_spawn_points` VALUES (16793708,'Alraune','Alraune',13,-250.881,-3.837,-531.278,105);
+INSERT INTO `mob_spawn_points` VALUES (16793709,'Coastal_Opo-opo','Coastal Opo-opo',12,-268.025,-3.569,-526.613,241);
+INSERT INTO `mob_spawn_points` VALUES (16793710,'Ignis_Fatuus','Ignis Fatuus',11,-281.653,-3.996,-522.317,251);
+INSERT INTO `mob_spawn_points` VALUES (16793711,'Peerifool','Peerifool',18,-321.315,-2.726,-741.241,132);
+INSERT INTO `mob_spawn_points` VALUES (16793712,'Peerifool','Peerifool',18,-322.777,-2.768,-739.350,12);
+INSERT INTO `mob_spawn_points` VALUES (16793713,'Peerifool','Peerifool',18,-321.871,-2.696,-742.227,152);
+INSERT INTO `mob_spawn_points` VALUES (16793714,'Peerifool','Peerifool',18,-322.847,-2.623,-741.673,67);
+INSERT INTO `mob_spawn_points` VALUES (16793715,'Peerifool','Peerifool',18,-322.066,-2.615,-738.434,116);
+INSERT INTO `mob_spawn_points` VALUES (16793716,'Peerifool','Peerifool',18,-320.565,-2.797,-740.059,140);
```
Comment (ibm2431) [inline sql/mob_spawn_points.sql:520]: Others: These Peerifool are [spawned simultaneously during One Good Deed](https://ffxiclopedia.fandom.com/wiki/Peerifool). They're not a group of mobs someone pulled to a different location.

**Review by ibm2431 — APPROVED** [review]
As always, you are my hero~

I also approve of the new QM convention.

**Review by TeoTwawki — APPROVED** [review]
I for one, welcome our new qm_overlords naming convention.

**Review by zach2good — APPROVED** [review]

**Pull request merged**